### DISCUSSION TITLE
Question about editing peaks / peak list (target)

### DISCUSSION THREAD

**Discussion by AnnaClo** [OpenChrom/openchrom#467] [Q&A] [answered]
After peak identification, I'm trying to edit the peaks in a saved .ocb file (generated by batch processing). I want to modify the Best target of some peaks. I enabled the table content modus in the peak list, this allows me to type in the Best target cell, however such change disappears if I press enter or another cell in the table. Is there a way to save such a change in peak name in the chromatogram? 

![image](https://github.com/OpenChrom/openchrom/assets/140986173/97e3391e-ccbf-482b-a12f-1732a78fa880)

**Comment by Mailaender** [comment, marked as answer]
The https://lablicate.com/marketplace/extensions/powertools have peak naming in that area. To do the same in `Data Analysis` you have open the `Targets` menu again. Enable the :pencil2: `edit` toolbar via the button and enter a new one or edit existing ones:

![image](https://github.com/OpenChrom/openchrom/assets/756669/6b148ee2-1e17-4f21-b694-95ceecf700d9)
